### DISCUSSION TITLE
Proper way to populate a lookup entity?

### DISCUSSION THREAD

**Discussion by deduced** [mikro-orm/mikro-orm#2061] [Q&A]
In my project, I  have created an entity "Category" that will act as  a lookup table  (https://github.com/thebrutalcorporation/spensa/blob/master/server/src/populateLookupTables.ts). The entity will be populated just once. 

The reason I created a script to populate is that I did not find a way to generate the SQL without writing it by hand in the migration file. Is there a better way to do this than my script, which I find to be clunky as it only needs to run once after the category table is created. 

Thanks!

**Comment by B4nan** [comment]
You could use seeder in v5:

https://mikro-orm.io/docs/next/seeding/
